modules/peers.js

```diff
@@ -29,7 +29,7 @@ let self;
 const __private = {};
 let definitions;
 
-const peerDiscoveryFrequency = 30000;
+const peerDiscoveryFrequency = 10000;
```

Comment:
So we discovered that its related to PRC max calls limit, then why we still decreasing this value. Twice a minute for a network discovery is a good value.

Reply:
Risk mitigation ;p but yeah it's definitely worth running another betanet test with 30s discovery.

Reply:
Note that it's not just about discovery though, it also fetches the latest status of peers based on that interval.